Components/ResposiveImages/ResponsiveImages.css

```diff
@@ -1,5 +1,6 @@
 .picture-information {
   margin-top: 0 !important;
+  margin-bottom: 0 !important;
```

Comment:
use shorthand

Reply:
btw cant use only for resetting top and bottom
https://developer.mozilla.org/en-US/docs/Web/CSS/Shorthand_properties#margin_and_padding_properties

Reply:
i dont want left right auto

Reply:
here 0 would be top-bottom and auto would be left right so

Reply:
ok cool! Go ahead!